src/example/App/VariableText.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import createReactClass from 'create-react-class';
 import {shouldComponentUpdate} from 'react/lib/ReactComponentWithPureRenderMixin';
```

Comment:
Deprecation on React v15.4.

> However, there is a possibility that you imported private APIs from react/lib/*, or that a package you rely on might use them. We would like to remind you that this was never supported, and that your apps should not rely on internal APIs. The React internals will keep changing as we work to make React better.
> -- [React v15.4.0 - React Blog](https://facebook.github.io/react/blog/2016/11/16/react-v15.4.0.html "React v15.4.0 - React Blog")

I've test react-height on react v16.

```
npm i -D react@next
```

`react/lib/ReactComponentWithPureRenderMixin` has been removed.
Because, React v16 switch to flat bundles. 

> Switch to flat bundles (no more react-dom/lib/*, internals are truly private) 
> -- https://github.com/facebook/react/issues/8854

![package 2 2017-04-18 09-39-33](https://cloud.githubusercontent.com/assets/19714/25109878/fecb35d8-241a-11e7-980b-68e48bedf6f1.png)